rpc/flipt/flipt.proto

```diff
@@ -110,6 +111,7 @@ message Flag {
   repeated Variant variants = 7;
   string namespace_key = 8;
   FlagType type = 9;
+  google.protobuf.Struct metadata = 10;
```

Comment:
Im seeing a lot of errors in tests again because the API clients and so on are expecting a non-nil metadata.
I think, accepting nil is something we probably can handle. It might work if you set each of these metadata fields as `optional`.

We could try that and see if it makes the tests happy.

```suggestion
  optional google.protobuf.Struct metadata = 10;
```